src/Symfony/Component/Notifier/Bridge/Discord/DiscordTransport.php

```diff
@@ -65,7 +67,7 @@ protected function doSend(MessageInterface $message): SentMessage
 
         $content = $message->getSubject();
 
-        if (\strlen($content) > 2000) {
+        if (mb_strlen($content) > self::MAX_SUBJECT_LENGTH) {
```

Comment:
calling mb_* without an explicit charset is something that should be banned in all situations.
It's a bit like gambling with charsets :)
`UTF-8` must be specified here (and only utf8 is supported, since we use json along the path to Discord)